### ISSUE TITLE
Add `Infinity` support to `prefer-number-properties` rule

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#863]
Fail:

```js
const foo = Infinity;
const bar = -Infinity;
```

Pass

```js
const foo = Number.POSITIVE_INFINITY;
const bar = Number.NEGATIVE_INFINITY;
```

I [mentioned this in the original proposal](https://github.com/sindresorhus/eslint-plugin-unicorn/issues/594#issuecomment-601560552), but [@sindresorhus prefer use `Infinity`](https://github.com/sindresorhus/eslint-plugin-unicorn/issues/594#issuecomment-601564678).

Can we add this support behind options?

**Comment by sindresorhus** [comment]
After thinking about it some more, I'm ok with doing this by default. It's good to reduce the use of globals. It should be noted though that they have the exact same behavior.

https://stackoverflow.com/questions/8522400/infinity-vs-number-positive-infinity

**Comment by papb** [comment]
Please still do it with an option, so I can disable only the part involving `Infinity`. I don't mind the default choice, I just want to be able to disable only this part.
